test/plugins/individual-collateral/stargate/constants.ts

```diff
@@ -1,25 +1,50 @@
 import { bn, fp } from '#/common/numbers'
 import { networkConfig } from '#/common/configuration'
+import hre from 'hardhat'
+import { useEnv } from '#/utils/env'
 
-export const STARGATE = networkConfig['1'].tokens['STG']!
-export const STAKING_CONTRACT = '0xB0D502E938ed5f4df2E681fE6E419ff29631d62b'
-export const SUSDC = networkConfig['1'].tokens['sUSDC']!
-export const SUSDT = networkConfig['1'].tokens['sUSDT']!
-export const SETH = networkConfig['1'].tokens['sETH']!
-export const USDC = networkConfig['1'].tokens['USDC']!
-export const USDT = networkConfig['1'].tokens['USDT']!
-export const USDC_HOLDER = '0x0a59649758aa4d66e25f08dd01271e891fe52199'
-export const USDC_USD_PRICE_FEED = networkConfig['1'].chainlinkFeeds['USDC']!
-export const ETH_USD_PRICE_FEED = networkConfig['1'].chainlinkFeeds['ETH']!
+const forkNetwork = useEnv('FORK_NETWORK') ?? 'mainnet'
+let chainId;
+
+switch(forkNetwork) {
+    case 'mainnet':
+        chainId = '1'
+        break
+    case 'base':
+        chainId = '8453'
+        break
+    default:
+        chainId = '1'
+        break
+}
+
+const USDC_NAME = chainId == '8453' ? 'USDbC' : 'USDC'
+const sUSDC_NAME = chainId == '8453' ? 'sUSDbC' : 'sUSDC'
+
+export const STARGATE = networkConfig[chainId].tokens['STG']!
+export const STAKING_CONTRACT = networkConfig[chainId].STARGATE_STAKING_CONTRACT!
+export const SUSDC = networkConfig[chainId].tokens[sUSDC_NAME]!
+export const SUSDT = networkConfig[chainId].tokens['sUSDT']!
+export const SETH = networkConfig[chainId].tokens['sETH']!
+export const USDC = networkConfig[chainId].tokens[USDC_NAME]!
+export const USDT = networkConfig[chainId].tokens['USDT']!
+export const USDC_HOLDER = chainId == '8453' ? '0x4c80e24119cfb836cdf0a6b53dc23f04f7e652ca' : '0x0a59649758aa4d66e25f08dd01271e891fe52199'
+export const USDC_USD_PRICE_FEED = networkConfig[chainId].chainlinkFeeds['USDC']! // currently same key for USDC and USDbC
+export const ETH_USD_PRICE_FEED = networkConfig[chainId].chainlinkFeeds['ETH']!
 export const SUSDC_POOL_ID = bn('1')
 export const WSUSDC_NAME = 'Wrapped S*USDC'
 export const WSUSDC_SYMBOL = 'wS*USDC'
 
+export const USDbC = networkConfig[chainId].tokens['USDbC']!
+export const SUSDbC = networkConfig[chainId].tokens['sUSDbC']!
```

Comment:
These two variables are not needed as thy are covered by. `USDC` and `SUSDC` I think? Which already are dynamic based on the network.